pyproject.toml

```diff
@@ -51,6 +56,8 @@ pytest-xdist = ">=2.4.0"
 Sphinx = ">=5.0"
 pytest-rerunfailures = ">=10.2"
 typeguard = ">=2.13"
+ifaddr = "^0.2.0"
```

Comment:
To be frankly honest, we do not need ifaddr. Our own testsuite can rely on testinfra, even if testinfra became optional.

Reply:
Well, then I can't run the test suite on Windows or OS X.

Reply:
Then please make this a windows & OSX only dependency